source/form.html

```diff
@@ -0,0 +1,157 @@
+      </a>
+      <button class="header__toggle" type="button">
+        <span class="visually-hidden">Открыть меню</span>
+        <svg xmlns="http://www.w3.org/2000/svg" width="24" height="18" fill="none"><path stroke="#000" stroke-width="1.5" d="M0 1.25h24M0 9.25h24M0 17.25h24"/></svg>
+        <div class="header__popover-menu">
+          <ul class="header__popover-menu__list">
+            <li class="header__popover-menu__item">
+              <a class="header__popover-menu__link">
+                Главная
+              </a>
+            </li>
+            <li class="header__popover-menu__item">
+              <a class="header__popover-menu__link">
+                Каталог продукции
+              </a>
+            </li>
+            <li class="header__popover-menu__item">
+              <a class="header__popover-menu__link header__nav--current">
+                Подбор программы
+              </a>
+            </li>
+          </ul>
+          </div>
+      </button>
+    </header>
+    <section class="main-form">
+      <div class="form__wrapper">
+        <h2 class="form__title">Подбор программы</h2>
+        <b class="form__intro">Заполните анкету, и мы подберем программу питания для вашего кота</b>
+        <form class="form">
+          <div class="form__data">
+            <div class="form__data__persona">
+              <div class="form__name">
+                <label class="form__name__item" for="name">Имя:*</label>
+                <input class="form__name__field" type="text" id="name" name="name" aria-label="Введите имя вашего кота" placeholder="Барсик" required>
```

Comment:
Тут тоже не надо уникальные классы задавать. Все ж одинаковые поля.
```suggestion
                <label class="form__label" for="name">Имя:*</label>
                <input class="form__input" type="text" id="name" name="name" aria-label="Введите имя вашего кота" placeholder="Барсик" required>
```